dapps/react-dapp-v2/src/contexts/ClientContext.tsx

```diff
@@ -154,7 +155,7 @@ export function ClientContextProvider({
           // Create a flat array of all requested chains across namespaces.
           const standaloneChains = Object.values(requiredNamespaces)
             .map((namespace) => namespace.chains)
-            .flat();
+            .flat() as string[];
```

Comment:
If you have other types suggestions, please let me know. 

I know we can use `SessionTypes.Struct.requiredNamespaces` but was unsure of how best to navigate it.